www/grpc/proto/wallet.proto

```diff
@@ -55,6 +55,9 @@ enum AddressType {
   ADDRESS_TYPE_VALIDATOR = 1;
   // Account address type with BLS signature scheme.
   ADDRESS_TYPE_BLS_ACCOUNT = 2;
+  // Account address type with Ed25519 signature scheme.
+  // Note: Generating a new Ed25519 address requires the wallet password.
+  ADDRESS_TYPE_ED25519_ACCOUNT = 3;
```

Comment:
@b00f For this type needs to get wallet password in the message `GetNewAddressRequest`?

Reply:
Yes, We need the password to derive a new address. The reason is explained  here: https://github.com/pactus-project/pactus/blob/0fd3ac211075b48abf8315e3a51d757c762066b3/crypto/ed25519/hdkeychain/extendedkey.go#L70

Reply:
Added the password to the `GetNewAddress` input. Please review again